<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsTable</class>
 <widget class="QWidget" name="ResultsTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="RenamableTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Global Columns:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="global_lines">
            <property name="text">
             <string>Lines</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="global_gates">
            <property name="text">
             <string>Gates</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="global_qc">
            <property name="text">
             <string>Quantum Costs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="global_tc">
            <property name="text">
             <string>Transistor Costs</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Extra Columns:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="configure_extra_columns">
            <property name="text">
             <string>Configure</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="extra_columns">
            <property name="text">
             <string>No extra columns</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Export:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="export_pdf">
            <property name="text">
             <string>Preview PDF</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="export_latex">
            <property name="text">
             <string>As LaTeX Table</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenamableTabWidget</class>
   <extends>QTabWidget</extends>
   <header>core/RenamableTabWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>global_lines</tabstop>
  <tabstop>global_gates</tabstop>
  <tabstop>global_qc</tabstop>
  <tabstop>global_tc</tabstop>
  <tabstop>configure_extra_columns</tabstop>
  <tabstop>export_pdf</tabstop>
  <tabstop>export_latex</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
